sovrin_client/test/cli/test_nym_suspension.py

```diff
@@ -105,7 +105,7 @@ def testTrusteeSuspendingTrustAnchor(be, do, trusteeCli, trustAnchorAdded,
        expect=nymAddedOut, mapper=trustAnchorAdded)
     s = SimpleSigner().identifier
     be(trustAnchorCli)
-    errorMsg = 'UnauthorizedClientRequest'
+    errorMsg = "UnauthorizedClientRequest('USER cannot add USER'"
```

Comment:
Maybe we can make an exception for this case where we replace `None cannot add None` by `Not permitted to add NYM`